gotocompany/optimus/core/v1beta1/job_spec.proto

```diff
@@ -224,7 +224,12 @@ message UpsertJobSpecificationsRequest {
 }
 message UpsertJobSpecificationsResponse {
   string log = 1;
-  repeated string successful_job_names = 2;
+  repeated UpsertJobStatus results = 2;
+}
+
+message UpsertJobStatus {
+  string job_name = 1;
+  string status = 2;
```

Comment:
any reason why we need status here? is `successful_job_names` not suffice?

Reply:
For UX reasons, want to highlight which jobs that are added/updated/skipped/failed. Let me know if you have any concern with this.

Reply:
got it, no concern. just keep a note, as the status is in `string`, other devs can have free form to put anything here

Reply:
@deryrahman on second thought, it is better to have the names so that we can easily register the successful specs through Shield proxy. thinking of having these instead:
- `successful_job_names` (we will not differentiate add/update)
- `skipped_job_names`
- `failed_job_names`
Wdyt?